instrumentation/opentelemetry-instrumentation-aiokafka/README.rst

```diff
@@ -0,0 +1,22 @@
+OpenTelemetry AIOKafka integration
+======================================
```

Comment:
This should match the number of chars of the line above

Reply:
done